template_multiplatform_components/lumin/tsconfig.json

```diff
@@ -1,7 +0,0 @@
-{
```

Comment:
we don't need this tsconfig file in this folder?

Reply:
We need this in template_overlay_typescript_components, template_multiplatform_components is by default without typescript support